src/bentoml/_internal/bento/bento.py

```diff
@@ -588,6 +588,14 @@ class BentoInfo:
     conda: CondaOptions = attr.field(factory=lambda: CondaOptions().with_defaults())
     envs: t.List[t.Dict[str, str]] = attr.field(factory=list)
 
+    @property
+    def get_models(self) -> t.List[BentoModelInfo]:
```

Comment:
```suggestion
    def get_models(self) -> t.List[BentoModelInfo]:
```

`get_xxx` is not a proper name for a property since it looks like an action. Recommend `all_models`

Reply:
yeah. make sense